contracts/exchange/test/balance_stores/local_balance_store.ts

```diff
@@ -0,0 +1,106 @@
+import { assetDataUtils } from '@0x/order-utils';
+import { AssetProxyId } from '@0x/types';
+import { BigNumber } from '@0x/utils';
+import * as _ from 'lodash';
+
+import { BalanceStore } from './balance_store';
+
+export class LocalBalanceStore extends BalanceStore {
```

Comment:
This is also super nice! I'm super happy about the fact that we removed the balance logic from the simulators, and it seems like it would be trivial to use this with `MatchOrders`